Sources/Core/ModernAVPlayer.swift

```diff
@@ -110,6 +111,16 @@ public final class ModernAVPlayer: NSObject, MediaPlayer {
         }
         context.loadMedia(media: media, autostart: autostart)
     }
+   
+    ///
+    /// Replaces the current item metadata with the new metadata
+    /// This method is dedicated for updating nowPlayingInfo data, and though, will discard duration and isLive previously set.
+    ///
+    /// - parameter metadata: metadata to load
+    ///
+    public func updateNowPlayingInfo(metadata: PlayerMediaMetadata) {
+        context.nowPlaying.update(metadata: metadata, duration: nil, isLive: true)
```

Comment:
isLive property is media attribute, not metadata like you said.
we may only change metadata by this method, not duration or isLive attributes.

then we can remove "...and though, will discard duration and isLive previously set." from the documentation. 
these attributes will be only set when media is loaded. 

Wdyt ?

Reply:
Ok, added a new function update(metadata:)